discord_bot/discord_bot.py

```diff
@@ -146,22 +153,37 @@ def load_config(config_file_path):
                     event = message["t"]
```

Comment:
My other comment got hidden, but we should probably move all of this dispatch/event handling into its own separate method to let `gateway_handler` be smaller.